test/dish_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+require 'pry'
+
+class DishTest < Minitest::Test
```

Comment:
GREAT tests!  Think about maybe testing more than one Dish, but this looks awesome